src/backend/opencl/wrappers/AdlLib_linux.cpp

```diff
@@ -130,9 +147,9 @@ AdlHealth xmrig::AdlLib::health(const OclDevice &device)
     AdlHealth health;
     health.clock        = sysfs_read(prefix + "freq1_input") / 1000000;
     health.memClock     = sysfs_read(prefix + "freq2_input") / 1000000;
```

Comment:
I'm not sure how these numbers were picked but the new function could be modified to accept a starting index for `i`, thoughts?